madgraph/various/banner.py

```diff
@@ -5293,7 +5293,12 @@ def default_setup(self):
         self.add_param('lpp1', 1, fortran_name='lpp(1)')        
         self.add_param('lpp2', 1, fortran_name='lpp(2)')                        
         self.add_param('ebeam1', 6500.0, fortran_name='ebeam(1)')
-        self.add_param('ebeam2', 6500.0, fortran_name='ebeam(2)')        
+        self.add_param('ebeam2', 6500.0, fortran_name='ebeam(2)')
+        
+##############Anton
+        self.add_param('asymm_choice', False, fortran_name='asymm_choice',comment='Can be True/False. Works only with reweight_scale or/and reweight_PDF flags = True ')
+##############Anton      
```

Comment:
Should not add a condition in check_consistency that if asymm_choice is on True, lhadpdf is on?
and that reweight_pdf is On also?

For the first condition, i guess it makes sense to crash in that case,
while for the second, you can switch it on with a warning/information